kodilla-jdbc/src/test/java/com/kodilla/jdbc/DbManagerTestSuite.java

```diff
@@ -0,0 +1,80 @@
+package com.kodilla.jdbc;
+
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class DbManagerTestSuite {
+
+    ////TEST Z KODILLI START
+    @Test
+    public void testConnect() throws SQLException {
+        //Given
+        //When
+        DbManager dbManager = DbManager.getInstance();
+        //Then
+        Assert.assertNotNull(dbManager.getConnection());
+    }
+
+    @Test
+    public void testSelectUsers() throws SQLException {
+        //Given
+        DbManager dbManager = DbManager.getInstance();
+
+        //When
+        String sqlQuery = "SELECT *from USERS";
+        Statement statement = dbManager.getConnection().createStatement();
+        ResultSet rs = statement.executeQuery(sqlQuery);
+
+        //Then
+        int counter = 0;
+        while (rs.next()) {
+            System.out.println(rs.getInt("ID") + ", " +
+                    rs.getString("FIRSTNAME") + ", " +
+                    rs.getString("LASTNAME"));
+            counter++;
+        }
+        rs.close();
+        statement.close();
+        Assert.assertEquals(5, counter);
+    }
+
+    ////KONIEC TESTU KODILLI
+
+    ////MÓJ TEST Z ZADANIA
+    @Test
+    public void shouldReturn2UsersHaving2posts()throws SQLException {
```

Comment:
wiem, że w zadaniu jest napisane, że powinieneś sprawdzić faktyczną ilość użykowników w bazie, ale w realnym świecie robi się to raczej na mockach bazy danych, żeby mieć pewność, że test faktycznie działa, a nie zadziałał akurat dla tego określonego stanu bazy danych